src/main.ts

```diff
@@ -410,17 +410,12 @@ export class Purchases {
 
   /**
    * Renders an RC Paywall and allows the user to purchase from it using Web Billing.
-   * @experimental
-   * @internal
```

Comment:
Just to check that we're ready then for this to be used by customers on the next release? Just confirming :)

Reply:
Yes, this is already being used in Web Purchase Links!